content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Enabling Rate Limits using Envoy
+description: This task shows you how to use Istio to dynamically limit the traffic to a service.
+weight: 10
+keywords: [policies,quotas]
+aliases:
+    - /docs/tasks/rate-limit.html
+owner: istio/wg-policies-and-telemetry-maintainers
+test: n/a
+---
+
+This task shows you how to use Envoy's native rate limiting in Istio to dynamically limit the traffic to a
+service.
+
+## Before you begin
+
+1. Setup Istio in a Kubernetes cluster by following the instructions in the
+   [Installation Guide](/docs/setup/getting-started/).
+
+1. Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+
+    The Bookinfo sample deploys 3 versions of the `reviews` service:
+
+    * Version v1 doesn’t call the `ratings` service.
+    * Version v2 calls the `ratings` service, and displays each rating as 1 to 5 black stars.
+    * Version v3 calls the `ratings` service, and displays each rating as 1 to 5 red stars.
+
+    You need to set a default route to one of the versions. Otherwise, when you send requests to the `reviews` service, Istio routes requests to all available versions randomly, and sometimes the output contains star ratings and sometimes it doesn't.
+
+1. Set the default version for all services to v1.
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/networking/virtual-service-all-v1.yaml@
+    {{< /text >}}
+
+## Rate limits
+
+In this task, you configure Istio to rate limit traffic to `productpage` whenever request is routed to a specific path.
+Rate limit using Envoy can be achieved using either Global Rate Limiting or Local Rate Limiting.
```

Comment:
```suggestion
Envoy supports both Global Rate Limiting and Local Rate Limiting.
```

Reply:
I would add two extra sentences explaining the difference between Global and Local, with links to the relevant Envoy docs if possible.

Reply:
Done. Added docs link in their specific sections